config/locales/en.yml

```diff
@@ -451,9 +451,37 @@ en:
     destroy: "Destroy"
     rename: "Rename"
 
+  components:
```

Comment:
This key already exists further down and is overridden there. App reports translation missing.

Reply:
Yep, right, this key has been added in the meantime.
I'm gonna change it, but it will probably need for me to squash commits ;)